superbench/runner/runner.py

```diff
@@ -449,9 +455,12 @@ def run(self):
                     if not mode.pattern:
                         ansible_rc = self._run_proc(benchmark_name, mode, {'proc_rank': 0})
                     else:
-                        with open(self._output_path / 'hostfile', 'r') as f:
-                            host_list = f.read().splitlines()
-                        pattern_hostx = gen_tarffic_pattern_host_group(host_list, mode.pattern)
+                        if not os.path.exists(self._output_path / 'hostfile'):
+                            host_list = []
```

Comment:
what if the hostfile does not exist, the benchmark won't run?

Reply:
Yes, added a warning log here.

Reply:
can you just `continue` then? seems no need to set [] and run following code

Reply:
updated.